quarkus/service/build.gradle.kts

```diff
@@ -165,6 +165,8 @@ tasks.named<Test>("intTest").configure {
   // Same issue as above: allow a java security manager after Java 21
   // (this setting is for the application under test, while the setting above is for test code).
   systemProperty("quarkus.test.arg-line", "-Djava.security.manager=allow")
+  // force the locale, just in case the user is having another default locale
+  systemProperty("quarkus.default-locale", "en_US")
```

Comment:
What is the issue if the locale is not `en_US`?
Don't we need this for `test` as well?

Reply:
IDK about what JB hit, but in general String upper/lower case conversion depends on the local even for the ASCII char sub-set

Reply:
In other words, I'd prefer to have this setting fixed in the main server properties, not just in test.

Are there reasons to run the Server under a custom locale?

Reply:
For instance https://github.com/apache/polaris/blob/main/integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisManagementServiceIntegrationTest.java#L366 and https://github.com/apache/polaris/blob/main/integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisManagementServiceIntegrationTest.java#L383 assumed the locale is `en_US`: it expects to return `must not be null`.

If the locale is not `en_US`, let say `fr_FR`, then the tests fail as we have `ne doit pas être nul` instead of `must not be null`.

Reply:
OTOH, don't we want to have i18n in Polaris? French customers may be happier reading error messages in French. Setting the locale to `en_US` sounds very... US centric :-) Can't we adapt the tests instead?

Reply:
i18n is extra work. I do not think merely changing the server's default locale is enough. For instance, given an error do we want to render it in one locale in the server log and in another locale for the REST/HTTP response (the client's locale)?

From my POV, if we allow custom locales we should forbid `String.toUpper/LowerCase()` in code.

Reply:
@adutra that's a good point, but I think there's more work to do for i18n support. Maybe we can do step by step. My PR is only in the integration tests here. So maybe we can start by this and create an issue for i18n.

Reply:
Also, we can fix string manipulation in Polaris code, but I would not be 100% sure about dependencies :)

Reply:
There is definitely more work than just that to get good i18n, but OTOH if we carve the locale in stone as `en_US`, that's not a good start ;-)

Reply:
Question: Since `quarkus.default-locale` is a Quarkus setting, why no put it in application.properties? I suspect all servers should behave the same way - even on French systems ;)